flang/test/Fir/fir-ops.fir

```diff
@@ -541,15 +541,15 @@ func @arith_real(%a : f128, %b : f128) -> f128 {
 // CHECK: [[VAL_175:%.*]] = subf [[VAL_174]], [[VAL_170]] : f128
 // CHECK: [[VAL_176:%.*]] = mulf [[VAL_173]], [[VAL_175]] : f128
 // CHECK: [[VAL_177:%.*]] = divf [[VAL_176]], [[VAL_169]] : f128
-// CHECK: [[VAL_178:%.*]] = fir.modf [[VAL_177]], [[VAL_170]] : f128
+// CHECK: [[VAL_178:%.*]] = remf [[VAL_177]], [[VAL_170]] : f128
```

Comment:
@schweitzpgi , given these are not FIR ops anymore, is there a point keeping all these built-in dialect op tests (they should be tested by MLIR regression tests) ?

I am also OK keeping them as trace of what we depend upon in FIR/Lowering.

Reply:
True. We shouldn't have to test the builtin dialect in FIR. It doesn't hurt anything, but we could argue to remove it. The interesting thing is that this somehow broke the builtin call op. So it's good we tested it. :)